src/setup-docker/templates/fabric-docker/commands-generated.sh

```diff
@@ -15,10 +15,8 @@ generateArtifacts() {
   <%_ ordererGroups.forEach((ordererGroup) => { _%>
 
   <% if(!global.capabilities.isV3) {%> 
-  printItalics "Generating genesis block for group <%= ordererGroup.name %>" "U1F3E0"
-  genesisBlockCreate "$FABLO_NETWORK_ROOT/fabric-config" "$FABLO_NETWORK_ROOT/fabric-config/config" "<%= ordererGroup.profileName %>"
-  <% } else { %> 
```

Comment:
This else condition was required correct? I feel we should have a log regarding system channel support

Reply:
no, we don't need it. This is the default way Fabric works, no need to provide this information in logs